src/assets/data/GuideData.ts

```diff
@@ -0,0 +1,99 @@
+interface Shortcut {
+	keys: string;
+	description: string;
+}
+
+interface Guide {
+	title: string;
+	shortcuts: Shortcut[];
+}
+
+export const guideData: Guide[] = [
```

Comment:
We need to be consistent here in terms of constant naming. For example, use `GUIDE_DATA`, signalling instantly that this is a constant.

Reply:
I took note and will make sure to replace the given constants with the caps convention (im afraid replacing all of them in this PR is a bit too much)